xml/System.ComponentModel/TypeDescriptionProvider.xml

```diff
@@ -1035,9 +1035,9 @@
         <Parameter Name="instance" Type="System.Object" Index="0" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="instance">To be added.</param>
+        <param name="instance">An instance of the type. Can be <see langword="null" /> if no instance was passed to the <see cref="T:System.ComponentModel.TypeDescriptor" />.</param>
         <summary>Returns a custom type descriptor for the given type or object.</summary>
-        <returns>To be added.</returns>
+        <returns>An <see cref="T:System.ComponentModel.ICustomTypeDescriptor" /> that can provide metadata for the type.</returns>
         <remarks>The objectType parameter is always valid, but the instance parameter may be <see langword="null" /> if no instance was passed to TypeDescriptor. The method should return a custom type descriptor for the object. If the method is not interested in providing type information for the object it should return base.</remarks>
```

Comment:
There is no `objectType` parameter for this overload, therefore `instance` cannot be null
```suggestion
        <remarks>The method should return a custom type descriptor for the object. If the method is not interested in providing type information for the object it should return base.</remarks>
```